Project/RAFinder/app/src/main/java/edu/rosehulman/rafinder/model/dummy/DummyData.java

```diff
@@ -14,167 +15,143 @@
 
     private static Resident[] room12 = { resmv, resmw, resmx, resmy, resmz };
 
+    private static final List<RoomEntry> rooms;
+    private static final List<DutyRosterItem> dutyRoster;
+
+    static {
+        rooms = Arrays.asList(
+                new RoomEntry("Lakeside", "101", room1),
+                new RoomEntry("Lakeside", "102", room2),
+                new RoomEntry("Lakeside", "103", room3),
+                new RoomEntry("Lakeside", "104", room4),
+                new RoomEntry("Lakeside", "105", room5),
+                new RoomEntry("Lakeside", "106", room6),
+                new RoomEntry("Lakeside", "107", room7),
+                new RoomEntry("Lakeside", "108", room8),
+                new RoomEntry("Lakeside", "109", room9),
+                new RoomEntry("Lakeside", "110", room10),
+                new RoomEntry("Lakeside", "111", room11),
+                new RoomEntry("Lakeside", "112", room12),
+                new RoomEntry("Lakeside", "113", room13));
+
+        dutyRoster = Arrays.asList(
+                new DutyRosterItem(new LocalDate(2015, 7, 10), RA1, RA2),
+                new DutyRosterItem(new LocalDate(2015, 7, 17), RA3, RA1),
+                new DutyRosterItem(new LocalDate(2015, 7, 24), RA2, RA3));
+    }
+
     public static List<DutyRosterItem> getDutyRoster() {
-        List<DutyRosterItem> dutyRoster = new ArrayList<DutyRosterItem>();
-        DutyRosterItem dutyRosterItem1 = new DutyRosterItem(new LocalDate(2015, 7, 10), RA1, RA2);
-        DutyRosterItem dutyRosterItem2 = new DutyRosterItem(new LocalDate(2015, 7, 17), RA3, RA1);
-        DutyRosterItem dutyRosterItem3 = new DutyRosterItem(new LocalDate(2015, 7, 24), RA2, RA3);
-        dutyRoster.add(dutyRosterItem1);
-        dutyRoster.add(dutyRosterItem2);
-        dutyRoster.add(dutyRosterItem3);
         return dutyRoster;
     }
 
     public static List<RoomEntry> getRooms() {
-        List<RoomEntry> rooms = new ArrayList<RoomEntry>();
-        rooms.add(new RoomEntry(room1, "BSB", "101"));
-        rooms.add(new RoomEntry(room2, "BSB", "102"));
-        rooms.add(new RoomEntry(room3, "BSB", "103"));
-        rooms.add(new RoomEntry(room4, "BSB", "104"));
-        rooms.add(new RoomEntry(room5, "BSB", "105"));
-        rooms.add(new RoomEntry(room6, "BSB", "106"));
-        rooms.add(new RoomEntry(room7, "BSB", "107"));
-        rooms.add(new RoomEntry(room8, "BSB", "108"));
-        rooms.add(new RoomEntry(room9, "BSB", "109"));
-        rooms.add(new RoomEntry(room10, "BSB", "110"));
-        rooms.add(new RoomEntry(room11, "BSB", "111"));
-        rooms.add(new RoomEntry(room12, "BSB", "112"));
-        rooms.add(new RoomEntry(room13, "BSB", "113"));
         return rooms;
-
-
     }
 
-    public static List<ResidentAssistant> getMyRAs() {
-        List<ResidentAssistant> ras = new ArrayList<ResidentAssistant>();
-        ras.add(resms);
-
-        return ras;
+    public static List<Employee> getMyRAs() {
```

Comment:
Generified type arguments in the next few methods. Didn't seem to affect the ListAdapters.